TriblerGUI/widgets/trustpage/index.html

```diff
@@ -34,6 +35,8 @@ <h2 class="title">
 
 <!-- MAIN -->
 <script src="js/style_config.js"></script>
+<script src="js/radial_view/radial_navigation.js"></script>
+<script src="js/radial_view/radial_stepping_animation.js"></script>
```

Comment:
Shouldn't this be `radial_stepping_animator`, to be consistent with the naming below?